Tools-Override/resources.targets

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this update should be moved into buildtools, but I believe there is a msbuild issue that needs to be fixed. I don't have a machine I can repro the task exec issue on though. MSBuild was trying to find a CLR v4 runtime to run the task on and I think this came from the defaulting logic here: https://github.com/Microsoft/msbuild/blob/6851538897f5d7b08024a6d8435bc44be5869e53/src/Build/Instance/TaskFactories/AssemblyTaskFactory.cs#L369